charts/pvc-autoresizer/templates/_helpers.tpl

```diff
@@ -60,3 +60,44 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the name of the webhook service
+*/}}
+{{- define "pvc-autoresizer.webhookService" -}}
```

Comment:
This definition was not reused and I thought it was overly. It would be nice if it could be rewritten to be simpler.

Reply:
It is used here 
https://github.com/tropnikovvl/pvc-autoresizer/blob/main/charts/pvc-autoresizer/templates/controller/mutatingwebhookconfiguration.yaml#L23

and here
https://github.com/tropnikovvl/pvc-autoresizer/blob/main/charts/pvc-autoresizer/templates/_helpers.tpl#L85